### ISSUE TITLE
Does the Instagram carousel send private information to curator.io or Instagram?

### ISSUE THREAD

**Issue by OsePedro** [blackettlabfam/website#19]
Apparently [even IP addresses are considered to be sensitive under GDPR laws](https://thehackernews.com/2022/01/german-court-rules-websites-embedding.html), so we should check whether or not the carousel complies. If it does send private information, we should either request the visitor's permission to do this, or find a way of creating a carousel that doesn't send private information.

**Comment by sophmrtn** [comment]
According to their [privacy policy](https://curator.io/privacy-policy), they only collect the IP address for logged-in users.

"curator.io also collects potentially personally-identifying information like Internet Protocol (IP) addresses for logged in users. curator.io only discloses logged in user IP addresses under the same circumstances that it uses and discloses personally-identifying information as described below."

They also state that  "curator.io uses cookies only for logged in users on the curator.io website to identify and track visitors, their usage of curator.io website, and their website access preferences.." again which sounds like it only applies to the logged-in user, i.e. the email account of the person who has registered with them and not visitors of our website.

**Comment by OsePedro** [comment]
Thanks for checking that @sophmrtn. It sounds like that section of the privacy policy only addresses people who have visited the curator.io website though, rather than visitors of other websites, like ours, that use the curator.io service (after all, it's under the "website visitors" section). The bit of the cookies section that's most relevant is the last sentence: "the curator.io service does not use cookies to deliver images to your website in any way" -- that's reassuring. The "Third Party Integrations / Service Providers" section says we must also agree to the [Facebook Platform Terms / Privacy Policy](https://developers.facebook.com/terms/dfc_platform_terms/), Facebook INC (1 Hacker Way, Menlo Park, CA 94025), but that only addresses how curator.io should treat the Instagram data that they access. So they don't seem to have declared that they _are_ collecting data about our visitors, which hopefully means that they're not doing so.

Having said this, I've just noticed that:

1. curator.io imposes a 2000 page views/month limit on the free account (is that what we're using?). [This video](https://help.curator.io/article/29-what-happens-when-i-reach-my-page-view-limit) says they stop pulling posts until the next month when we hit the limit. It's not clear to me if this means that visitors will still see old posts when we hit the limit, or if it means that they won't see any posts. For reference, our homepage had this many hits each month from Sept-Feb: 1170, 1548, 1011, 994, 1048, 1042. So we're not hitting the limit yet, but we might hit it soon.
2. The services I'm using to measure visitor stats _do_ collect IP addresses, to detect distinct visits (the data is only stored on my web server -- it's not sent elsewhere). I'll look up whether or not we need the visitor's consent to do this.

**Comment by sophmrtn** [comment]
@OsePedro: 

> 1. curator.io imposes a 2000 page views/month limit on the free account (is that what we're using?). [This video](https://help.curator.io/article/29-what-happens-when-i-reach-my-page-view-limit) says they stop pulling posts until the next month when we hit the limit. It's not clear to me if this means that visitors will still see old posts when we hit the limit, or if it means that they won't see any posts. For reference, our homepage had this many hits each month from Sept-Feb: 1170, 1548, 1011, 994, 1048, 1042. So we're not hitting the limit yet, but we might hit it soon. 

Yes we are on the free account. And from the docs and video, it means that visitors won't see the latest posts. It updates every 8 hours to fetch the most current ones so if we hit the limit then it'll stop fetching. I think for now this is fine, we don't actually need it to be up-to-date, it just serves as a nice way to integrate our posts on the site and give people an easy way to get to it.

> 2. The services I'm using to measure visitor stats do collect IP addresses, to detect distinct visits (the data is only stored on my web server -- it's not sent elsewhere). I'll look up whether or not we need the visitor's consent to do this.

Oh interesting, okay I'll leave that with you and then we can probably close this issue.
